capstone/requirements.in

```diff
@@ -89,6 +89,6 @@ mincss       # remove unused css styles
 requests  # also via aws-xray-sdk, coreapi, docker, moto, response
 
 # Elasticsearch
-elasticsearch-dsl
-django-elasticsearch-dsl
-django-elasticsearch-dsl-drf
+elasticsearch-dsl==6.4.0
```

Comment:
We don't want to specify versions in requirements.in -- that's what requirements.txt is for.

The simplest way to upgrade packages is with pip-compile's `-P` flag -- in this case, that would be `fab pip-compile:"-P elasticsearch-dsl django-elasticsearch-dsl django-elasticsearch-dsl-drf"`. At this point you could also just roll back the changes to requirements.in, which has the same effect.

Reply:
Done